src/packages/sdk/src/utils/loadEnv.ts

```diff
@@ -16,97 +17,63 @@ type CLIArgs =
       '--telemetry-information': StringConstructor
     }>
 
-interface DotenvResult {
-  error?: Error
-  parsed?: {
-    [name: string]: string
-  }
-}
+type DotenvResult = dotenv.DotenvConfigOutput & {
+  ignoreProcessEnv?: boolean | undefined;
+};
 
 interface LoadEnvResult {
   message: string
-  dotenvResult: DotenvResult
+  path: string
+  dotenvResult:  DotenvResult
 }
-
+/**
+ * Tries load env variables
+ *  1. Load .env from project root
```

Comment:
The order in the code seems different? It seems like "Load .env from project root" happens last?

Reply:
Yeah this needs to be updated I though I had